src/App.js

```diff
@@ -3,25 +3,52 @@ import './App.css';
 import Counter from './component/Counter/Counter';
 import SearchForm from './component/SearchForm/SearchForm';
 import GenreSelect from './component/GenreSelect/GenreSelect';
+import MovieTile from './component/MovieTile/MovieTile';
+import MovieDetails from './component/MovieDetails/MovieDetails';
+import movies from './temp/Movies.json';
+import SortControl from './component/SortControl/SortControl';
 
 function App() {
   const genres = ['Action', 'Comedy', 'Drama', 'Horror', 'Science Fiction'];
+  
   const [selectedGenre, setSelectedGenre] = useState('Action');
-
   const handleGenreSelect = (genre) => {
     console.log("Selecting " + genre)
     setSelectedGenre(genre);
   };
 
+  const sortFilters = ['RELEASE DATE', 'TITLE'];
```

Comment:
Move them to constant file , appConstants.js

Reply:
Fixed as part of 5ea7447